src/gui/torrentcontentmodel.cpp

```diff
@@ -375,10 +381,22 @@ Qt::ItemFlags TorrentContentModel::flags(const QModelIndex &index) const
 
 QVariant TorrentContentModel::headerData(int section, Qt::Orientation orientation, int role) const
 {
-    if ((orientation == Qt::Horizontal) && (role == Qt::DisplayRole))
+    if (orientation != Qt::Horizontal)
+        return {};
+
+    switch (role) {
+    case Qt::TextAlignmentRole: {
```

Comment:
Do we want to sort the cases alphabetically? Personally I think it is a good idea.

Reply:
Seems braces aren't required in this case.

Reply:
> Do we want to sort the cases alphabetically (with the exception of default: case placed last) ? Personally I think it is a good idea.

I sorted them based on the length of the block It was done this way `data()` too

> Seems braces aren't required in this case.

this is multiline I think there was a rule to wrap these in braces (I can be wrong)

also, nothing personal or don't want to sound rude but don't you guys nitpick a lot? PS: I don't have any problem applying the above changes though.

Reply:
>also, nothing personal or don't want to sound rude but don't you guys nitpick a lot?

All our written and unwritten coding style rules are designed to facilitate further code maintenance. If any requested change is even slightly able to facilitate future review or search for errors, then this is not a nitpick.

Reply:
>this is multiline I think there was a rule to wrap these in braces (I can be wrong)

There is a rule for `if-else`. Switch case only require braces if there are variable declarations in it (although I would avoid such cases by restructuring the code).